server/src/main/java/io/seata/server/session/SessionHolder.java

```diff
@@ -89,15 +89,19 @@ public static void init(String mode) throws IOException {
             mode = CONFIG.getConfig(ConfigurationKeys.STORE_MODE);
         }
         //the store mode
-        StoreMode storeMode = StoreMode.valueof(mode);
+        StoreMode storeMode = StoreMode.get(mode);
         if (StoreMode.DB.equals(storeMode)) {
             //database store
-            ROOT_SESSION_MANAGER = EnhancedServiceLoader.load(SessionManager.class, StoreMode.DB.name());
-            ASYNC_COMMITTING_SESSION_MANAGER = EnhancedServiceLoader.load(SessionManager.class, StoreMode.DB.name(),
+            ROOT_SESSION_MANAGER =
+                    EnhancedServiceLoader.load(SessionManager.class, StoreMode.DB.getName());
```

Comment:
Try to keep the format as original as possible and reduce modification.

Reply:
OK